generators/client/templates/react/src/main/webapp/app/reducers/account.ts.ejs

```diff
@@ -54,20 +73,22 @@ export default (state = initialState, action) => {
 
 const apiUrl = '/api/account';
 // Actions
-export const saveAccountSettings: ICrudPutAction = account => ({
-  type: ACTION_TYPES.UPDATE_ACCOUNT,
-  meta: {
-    successMessage: messages.DATA_CREATE_SUCCESS_ALERT,
-    errorMessage: messages.DATA_UPDATE_ERROR_ALERT
-  },
-  payload: axios.post(apiUrl, account)
-});
+export const savePassword = (currentPassword, newPassword) => dispatch => {
```

Comment:
for simple actions no need to use dispatch just return the payload object
```
export const savePassword = (currentPassword, newPassword) => ({
  type: ACTION_TYPES.UPDATE_PASSWORD,
  payload: axios.post(`${apiUrl}/change-password`, { currentPassword, newPassword })
});
```

Dispatch is required only if you need to dispatch multiple actions or when having to do some pre/post processing